blueprints/app/files/package.json

```diff
@@ -58,7 +58,7 @@
     "ember-qunit": "^7.0.0",
     "ember-resolver": "^10.1.0",
     "ember-source": "~5.0.0",
-    "ember-template-lint": "^5.10.1<% if (welcome) { %>",
+    "ember-template-lint": "^5.10.3<% if (welcome) { %>",
```

Comment:
It is probably fine to bump this but I am trying to reconcile it with your commit message of "How did ember-template-lint get bumped?"

Reply:
The only thing I can think of is that I _accidentally_ applied a find and replace, where I thought I cancelled it.

Reply:
I just pushed a rename of the commit with mentioning that 5.10.2 had a bug

Reply:
That follow up commit is the one that did the bumping though...